tools/src/main/java/org/apache/kafka/tools/ProducerPerformance.java

```diff
@@ -46,6 +47,11 @@
 public class ProducerPerformance {
 
     public static void main(String[] args) throws Exception {
+        ProducerPerformance perf = new ProducerPerformance();
+        perf.start(args);
+    }
+    
+    public void start(String[] args) throws IOException {
```

Comment:
Could you change it from public to package-private?

Reply:
> Could you change it from public to package-private?

Done